R/promptAndRun.R

```diff
@@ -44,15 +51,16 @@ promptAndRun <- function(mydir = ".", user = NULL, daysback = 3) {
       myruns <- c(myruns, coupled) # add coupled paths
     }
     myruns <- myruns[file.exists(myruns)] # keep only existing paths
-    myruns <- unique(myruns[!is.na(myruns)])
+    myruns <- sort(unique(myruns[!is.na(myruns)]))
 
     if (length(myruns) == 0) {
       return("No runs found for this user. To change the reporting period (days) of the tool you need to specify also a user, e.g. rs2 -cr USER 1")
     } else {
       message("")
       message("Type 'rs2 -cr username DAYS' with DAYS an integer denoting how many days you want results from")
       message("")
-      message("Found these runs")
+      message("Found ", length(myruns), if (mydir == "-ar") " active", " runs.",
```

Comment:
I like the funny statement and thought it would be worth bringing it back :)

Reply:
fine with me, I pushed changes with the correct calculation :)